analytics_data_api/v0/views/course_summaries.py

```diff
@@ -154,12 +182,27 @@ def postprocess_field_dict(self, field_dict):
             # don't do expensive looping for programs if we are just going to throw it away
             field_dict = self.add_programs(field_dict)
 
+        if self.recent_date and 'recent_count_change' not in self.exclude:
+            field_dict = self.add_recent_count_change(field_dict)
+
         for field in self.exclude:
             for mode in field_dict['enrollment_modes']:
                 _ = field_dict['enrollment_modes'][mode].pop(field, None)
 
         return field_dict
 
+    def add_recent_count_change(self, field_dict):
+        # Pick the most recent daily count on-or-before the desired date
+        # If there are no daily counts before the desired date, assume 0
+        recents = models.CourseEnrollmentDaily.objects\
```

Comment:
It's cool if the linter if happy, but how I would probably tweak:

```
            recents = models.CourseEnrollmentDaily.objects.filter(
                course_id=field_dict['course_id'],
                date__lte=self.recent_date
            ).order_by('-date')[0:1]
```

Reply:
I'll try it, I agree it's more readable